source/content/guides/multisite/03-config.md

```diff
@@ -68,20 +68,43 @@ Make sure [Terminus](/terminus) is installed and [authenticated](/terminus/insta
 
 1. Open the `code` folder in your SFTP client, and download your site's `wp-config.php` file.
 
-1. Locate the configuration added by WP-CLI, and *modify* the line that sets `DOMAIN_CURRENT_SITE` from a hardcoded URL to a dynamic URL `$_SERVER['HTTP_HOST']`. This automatically detects the URL in each environment. You must replace this variable. For example:
-
+1. Locate the configuration added by WP-CLI, and *modify* the line that sets `DOMAIN_CURRENT_SITE` to a hardcoded URL. You will want to default to a dynamic URL (`$_SERVER['HTTP_HOST']`), however some environments on Pantheon (notably workflows that run containerized versions of WP-CLI like search and replace) will not have a `$_SERVER` global. For this reason, we recommend using the following configuration that provides fallbacks if `$_SERVER['HTTP_HOST']` is unavailable.
```

Comment:
> containerized versions

I think it'd be clearer and more accurate to say non-web requests. web requests also hit WordPress running in a container.

> will not have a `$_SERVER` global
Is that true? Or are they just missing `$_SERVER['HTTP_HOST']`

Reply:
> > will not have a $_SERVER global
> Is that true? Or are they just missing $_SERVER['HTTP_HOST']

I haven't verified 100% (because I don't know that there's a way to jump into those environments and run commands outside of running workflows, so we'd basically need to break something existing to test or write a lot of new Ygg internals code to make something new for testing), but I'm relatively sure this is accurate. However, we can be more specific and _just_ say that HTTP_HOST is not set, because that's the part that really matters.

Reply:
I've updated to be clearer about _just_ talking about `$_SERVER['HTTP_HOST']` (and not `$_SERVER` more generally) and updated the language to use "web requests"/"non-web requests".